_posts/2022-08-11-Agroserv.md

```diff
@@ -0,0 +1,17 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: We are part of the Horizon Europe project AgroServ
+tags:
```

Comment:
some tags from [this list](https://github.com/usegalaxy-eu/website#labels-for-posts-and-events)?